### ISSUE TITLE
Typing with the Element Web tab focused doesn't focus on the composer anymore

### ISSUE THREAD

**Issue by babolivier** [vector-im/element-web#19479]
Which greatly screws up my muscle memory :(

I've noticed it not doing this anymore since a day or two ago.

Element version: f550d63198ee-react-12ad01db5145-js-0c47e27f9fc7
Olm version: 3.2.3

**Comment by babolivier** [comment]
Reopening since https://github.com/vector-im/element-web/issues/19469 is apparently fixed but I'm still observing this issue on latest develop.element.io

**Comment by dbkr** [comment]
Can you reproduce the correct behaviour on /app? I tried but focusing something else & typing doesn't focus the composer. I have noticed I've been typing without the composer having focus a couple of times though, so I there must be something here. I'd say focusing the composer when the app/tab regains focus, but app doesn't do that either.

**Comment by babolivier** [comment]
I can also observe this behaviour on /app. When I'm not focused on the composer and start typing it doesn't refocus on the composer like it used to.

To add more context, this is a bug I've noticed about 3w ago (judging by the date of the initial report and my recollection that I had observed this change a couple of days before reporting it). Before that even if my focus wasn't on the composer if I started typing it would focus back on it. I assumed this was the initial intended behaviour since that's how it used to work, and Slack etc do the same.

**Comment by Bubu** [comment]
I've come across this in the latest release as well. I'm not 100% sure what the exact behaviour should be but one thing I noticed is that when you hit the "down-to-bottom"-arrow button, the composer explicitly loses focus. This seems like this is the wrong this to do, but maybe this was always how it was, just never a problem before because of the automatically giving it focus again while typing.

**Comment by chutten** [comment]
STR: 
1) Click in a blank space in the UI (Focus on compose window is lost)
2) Type.

Expected: focus would jump to the compose input and the typed character would appear there. Or focus would remain on the compose input when clicking on parts of the UI that don't take focus.

Actual: focus doesn't appear to change, and the typed character is consumed by default handlers (`/` will start "Quick Find", most letter keys do nothing, etc.)

I can confirm this is happening for me and at least one other at Mozilla with chat.mozilla.org. Which is a bit of a bummer.

**Comment by t3chguy** [comment]
Regression from Threads work
